From c99216d54b77aa0f5e60662cdd170b789c8185c9 Mon Sep 17 00:00:00 2001
From: Hubert Rzezniczak <hubert.rzezniczak@gmail.com>
Date: Tue, 13 Oct 2015 13:17:43 +0200
Subject: [PATCH 05/13] ALOGE does not match triggering condition.

Meeting the condition
if (sceneModeOverrides.count !=
                availableSceneModes.count * kModesPerSceneMode) {
results with ALOGE which describes the missmatch.
The description does not match the condition.

Change-Id: I1afc1ed8bb2ac8bdeda3eaf4dc8d3a5ff274cc3a
Signed-off-by: Hubert Rzezniczak <hubert.rzezniczak@gmail.com>
---
 services/camera/libcameraservice/api1/client2/Parameters.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/camera/libcameraservice/api1/client2/Parameters.cpp b/services/camera/libcameraservice/api1/client2/Parameters.cpp
index 4f4cfb0..8f7fc42 100644
--- a/services/camera/libcameraservice/api1/client2/Parameters.cpp
+++ b/services/camera/libcameraservice/api1/client2/Parameters.cpp
@@ -1011,7 +1011,7 @@ status_t Parameters::buildFastInfo() {
             ALOGE("%s: Camera %d: Scene mode override list is an "
                     "unexpected size: %zu (expected %zu)", __FUNCTION__,
                     cameraId, sceneModeOverrides.count,
-                    availableSceneModes.count);
+                    availableSceneModes.count * kModesPerSceneMode);
             return NO_INIT;
         }
         for (size_t i = 0; i < availableSceneModes.count; i++) {
-- 
2.5.0

